src/use_case/collect_recipe/CollectRecipeInputData.java

```diff
@@ -1,18 +1,17 @@
 package use_case.collect_recipe;
 
 public class CollectRecipeInputData {
```

Comment:
why do you add collect recipe in opened folder view model, the collect button should in the opened recipe view

Reply:
Hi, I do not see where the "opened folder view model" you are referring to. I did implement collect button should in RecipeView